### ISSUE TITLE
Update Acitiverecord

### ISSUE THREAD

**Issue by Nittarab** [lowjoel/activerecord-userstamp#30]
Hi, i'll try to use this gem with rails 5.1.4 but I get this. Is possible to update activerecord?
```
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    activerecord-userstamp was resolved to 3.0.4, which depends on
      activerecord (~> 4.2)

    rails (~> 5.1.4) was resolved to 5.1.4, which depends on
      activerecord (= 5.1.4)
```

**Comment by johnnyshields** [comment]
Please fix and release this gem thanks!

**Comment by mdelpozo** [comment]
Still having the same problem:

Resolving dependencies......

Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    activerecord-userstamp was resolved to 3.0.4, which depends on
      activerecord (~> 4.2)

    rails (~> 5.2.4) was resolved to 5.2.4, which depends on
      activerecord (= 5.2.4)

**Comment by Nittarab** [comment]
@mdelpozo @johnnyshields 
The gem is not updated in the ruby gems repository. 
you can use the GitHub version using this line in Gemfile

`gem 'activerecord-userstamp', git: 'https://github.com/lowjoel/activerecord-userstamp'`
